### PR TITLE
[DM-34613] Update for latest Gafaelfawr

### PR DESCRIPTION
**Pull request by rra**
- Bump version of Gafaelfawr and run it from a branch on dev and minikube for now
- Update Gafaelfawr chart for the new LDAP configuration structure
- Add `ldap-password` secret for Gafaelfawr
- Add LDAP and Firestore configuration for Gafaelfawr on IDF dev
- Stop specifying UIDs for mobu on data-dev
- Rename all mobu users to start with `bot-` and be more informative about what flock they're running
- Rename all other Gafaelfawr service users to start with `bot-`

### CODE REVIEW

File: installer/generate_secrets.py [hunk 1]
```diff
@@ -221,6 +221,11 @@ def _gafaelfawr(self):
                 f"Invalid gafaelfawr cloudsql value {use_cloudsql}"
             )
 
+        self.input_field("gafaelfawr", "ldap", "Use LDAP? (y/n):")
```
Comment (athornton) [inline installer/generate_secrets.py:224]: Should the use-cilogon-or-github go in the else clause, or do we still need it even if we're using LDAP?

**Review by athornton — APPROVED** [review]
Other than probably-my-lack-of-understanding with my comment about whether we still need cilogon-or-github, this looks good.

File: installer/generate_secrets.py (hunk 1 above)
Comment (rra) [reply installer/generate_secrets.py:224]: They're independent; you can use any authentication source and then still use LDAP for data (although in practice we will probably only use LDAP for the CILogon and OpenID Connect cases).

**Comment by rra** [comment]
Yes, indeed, thank you!

**Pull request merged**